src/env.h

```diff
@@ -73,14 +73,26 @@ namespace loader {
 class ModuleWrap;
 
 struct PackageConfig {
-  enum class Exists { Yes, No };
-  enum class IsValid { Yes, No };
-  enum class HasMain { Yes, No };
-
-  Exists exists;
-  IsValid is_valid;
-  HasMain has_main;
-  std::string main;
+  struct Exists {
+    enum Bool { No, Yes };
+  };
```

Comment:
Remind me again why these aren't just regular bool?

Reply:
I don't remember exactly, but I believe the pattern was established on master with feedback from @bnoordhuis.

Reply:
We could change these back to bools, but perhaps that can be a separate refactoring rather. For now I've stuck with the current pattern.